dependencies.list

```diff
@@ -1,4 +1,4 @@
 REALM_CORE_VERSION=5.1.2
-REALM_SYNC_VERSION=2.3.0
+REALM_SYNC_VERSION=3.0.0-alpha.2
```

Comment:
It's worth updating this (and the core version) to the most recent version before merging this.

Reply:
I will do that.